packages/cli/commands/theme.js

```diff
@@ -8,10 +9,10 @@ exports.command = 'theme';
 exports.describe = i18n(`${i18nKey}.describe`);
 
 exports.builder = yargs => {
-  addConfigOptions(yargs, true);
-  addAccountOptions(yargs, true);
```

Comment:
Should be safe to remove these here I think - we don't need them for `generate-selectors` and they're present a second time in the exports.builder of  `theme/marketplace-validate.js`